ivy_tests/test_ivy/test_functional/test_core/test_linalg.py

```diff
@@ -810,24 +805,25 @@ def test_svd(
     fw,
     fm,
 ):
-    if "float16" in input_dtype:
+    dtype, x = dtype_x
+    try:
```

Comment:
debug code? We should not be allowing a failing test to pass.

Reply:
Same, Darshan will update this later.

Reply:
Ah, there is a problem, when the dtype is float16, which is not supported,
there will not be any return values (line 810)
but as you can see there are some operations (line 829) later which are based on these return values

Reply:
Got it, we can instead catch the return with a single parameter, and then unpack the tuple if it is not none. If it is none, we can return. Make sense?

Reply:
Cool, this sounds great.